flink-streaming-java/src/main/java/org/apache/flink/streaming/api/datastream/WindowedStream.java

```diff
@@ -121,6 +124,23 @@ public WindowedStream(KeyedStream<T, K> input,
 	}
 
 	/**
+	 * Sets the allowed lateness. If the {@link WindowAssigner} used
+	 * is in processing time, then the allowed lateness is set to 0.
+	 */
+	@PublicEvolving
+	public WindowedStream<T, K, W> setAllowedLateness(Time lateness) {
```

Comment:
This should follow the same pattern as the other methods (`trigger()`, `evictor()`...), i.e. `allowedLateness(Time)`.